pkgs/development/python-modules/neo/default.nix

```diff
@@ -30,11 +32,16 @@ buildPythonPackage rec {
     quantities
   ];
 
-  nativeCheckInputs = [ nose ];
+  nativeCheckInputs = [
+    pytestCheckHook
+    pillow
+    which
+  ];
 
-  checkPhase = ''
-    nosetests --exclude=iotest
-  '';
+  disabledTestPaths = [
+    # No such file or directory: '/homeless-shelter/hdf5_plugin_path_maxwell'
```

Comment:
Can you try adding a `preCheck` hook with `export HOME=$(mktemp -d)`? Usually that fixes these errors.

Reply:
It works.... but now it tries to fetch something for the internet :<
```
FAILED neo/test/rawiotest/test_maxwellrawio.py::TestMaxwellRawIO::test_read_all - urllib.error.URLError: <urlopen error [Errno -3] Temporary failure in name ...
```